packages/ocaml-r/ocaml-r.0.1.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "philippe.veber@gmail.com"
+authors: ["Guillaume Yzyquel" "Maxence Guesdon" "Philippe Veber"]
+homepage: "https://github.com/pveber/ocaml-r"
+bug-reports: "https://github.com/pveber/ocaml-r/issues"
+license: "GPL"
+dev-repo: "https://github.com/pveber/ocaml-r.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "base" {build}
+  "configurator" {build}
+  "jbuilder" {build}
+  "ocamlfind"
```

Comment:
Do you still require ocamlfind ? I don't see any reference to `ocamlfind` or `Findlib` in your repository.

Reply:
Indeed I don't!